src/main/groovy/com/codebrig/arthur/generator/SchemaGenerator.groovy

```diff
@@ -162,7 +163,7 @@ class SchemaGenerator {
                     failCount.getAndIncrement()
                 }
             }
-        }).count()
+        }).collect(Collectors.counting())
```

Comment:
What's your reasoning behind this change? Aren't they equivalent?

Reply:
@BFergerson Somehow in my local it strangely doesn't execute anymore with `.count()`. You don't have this same issue?